solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1318,35 +1320,6 @@ describe("RandomBeacon - Group Creation", () => {
                 })
               }
             )
```

Comment:
I removed it because of two reasons:
- after some thought, it feels that we checked the behavior of array implementation itself, but not the functionality of the function. The only validation aspect of it was an indication that we're aware of a such scenario
- when it was used to calucate hash during the dkg submission we didn't have any other checks. But now since the members hashing moved to the dkg challenge, we have a test that validates misbehaved members order [here](https://github.com/keep-network/keep-core/blob/passing-members-hash/solidity/random-beacon/test/DKGValidator.test.ts#L407). If the misbehaved members are unordered, it won't even hit the `validateMembersHash`

Reply:
That's cool we still got it covert indirectly. I just wanted to make sure that we have a test to verify that it will blow up in case of mixed order.